Resources/Prototypes/_White/Entities/Clothing/OuterClothing/coats.yml

```
@@ -36,3 +36,14 @@
         Heat: 0.8
+
+- type: entity
+  parent: ClothingOuterBase
+  id: ClothingOuterVestCoatHop
+  name: HoP's coat
+  description: This coat was specifically designed for head of personnel.
+  components:
+  - type: Sprite
+    sprite: _White/Clothing/OuterClothing/Coats/hopcoat.rsi
+  - type: Clothing
+    sprite: _White/Clothing/OuterClothing/Coats/hopcoat.rsi
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)